js/apps/admin-ui/src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -92,18 +93,31 @@ export default function IdentityProvidersSection() {
 
   useFetch(
     async () => {
-      const provider = await adminClient.realms.findOne({ realm });
-      if (!provider) {
-        throw new Error(t("notFound"));
+      const providers = await adminClient.identityProviders.find();
+      if (!providers) {
+        throw new Error(t("common:notFound"));
       }
-      return provider.identityProviders!;
+      return providers;
     },
     (providers) => {
       setProviders(sortBy(providers, ["config.guiOrder", "alias"]));
     },
-    [key],
+    [],
   );
 
+  const loader = async (first?: number, max?: number, search?: string) => {
+    const params: IdentityProvidersQuery = {
+      first: first!,
+      max: max!,
+    };
+    if (search) {
+      params.search = search;
+    }
+    const providers = await adminClient.identityProviders.find({ ...params });
```

Comment:
There should be no need to spread the `params` here.
```suggestion
    const providers = await adminClient.identityProviders.find(params);
```